src/pages/trails/ballots/view/BallotView.vue

```diff
@@ -419,7 +425,7 @@ export default {
                             span.statistics-title Request amount
                         div.statistics-body(v-if="ballot.proposal_info")
                             span.text-weight-bold {{ getRequestAmountRounded(ballot.proposal_info.total_requested) }}&nbsp
-            .col-xs-12.col-sm.popup-right-col-wrapper
+            .col-xs-12.col-sm.popup-right-col-wrapper.cosoooooooo
```

Comment:
class name (?)  mistake?

Reply:
That would be "thiiiiiiiiing" in spanish. It was a mark I forgot to remove. My bad.
Fixed here [0d2041b](https://github.com/telosnetwork/app-telos-native/commit/0d2041b265080d7a9d821112a953977fa9688ae6)